agent/wsclient/client.go

```diff
@@ -161,14 +182,22 @@ func (cs *ClientServerImpl) Connect() error {
 		seelog.Warnf("Error creating a websocket client: %v", err)
 		return fmt.Errorf(string(resp) + ", " + err.Error())
 	}
-	cs.Conn = websocketConn
+	cs.conn = websocketConn
 	return nil
 }
 
+func (cs *ClientServerImpl) IsConnected() bool {
+	return cs.conn != nil
```

Comment:
To clarify -- `Disconnect()` closes the `net.Conn` object used by the websocket client, but it doesn't set it to nil.